pkgs/applications/misc/mupdf/default.nix

```diff
@@ -48,9 +48,12 @@ in stdenv.mkDerivation rec {
   # Use shared libraries to decrease size
   buildFlags = [ "shared" ];
 
-  makeFlags = [ "prefix=$(out) USE_SYSTEM_LIBS=yes" ];
+  makeFlags = [ "prefix=$(out)" "USE_SYSTEM_LIBS=yes" ]
+              ++ lib.optionals (!enableX11) [ "HAVE_X11=no" ]
+              ++ lib.optionals (!enableGL) [ "HAVE_GLUT=no" ];
```

Comment:
```suggestion
    ++ lib.optionals (!enableX11) [ "HAVE_X11=no" ]
    ++ lib.optionals (!enableGL) [ "HAVE_GLUT=no" ];
```

Reply:
@SuperSandro2000 I applied `nixpkgs-fmt` on the whole file.